src/main/scala/io/iohk/ethereum/jsonrpc/JsonMethodsImplicits.scala

```diff
@@ -38,7 +38,7 @@ trait JsonMethodsImplicits {
     }.toEither.left.map(_ => InvalidParams())
 
   private def decode(s: String): Either[JsonRpcError, Array[Byte]] = {
-    if(!s.startsWith("0x")) Left(InvalidParams())
+    if(!s.isEmpty && !s.startsWith("0x")) Left(InvalidParams())
```

Comment:
The strange thing is that geth doesn't seem to be consistent with asking or not for the prefix, as for example it doesn't allow the request `{"jsonrpc":"2.0","method":"eth_getBalance","params": ["c371cfecae774a728681c164360df9cec2a0a412", "0x1234"],"id":1}` (address without "0x" prefix).